tools/hive-bigquery/constants.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Google LLC
```

Comment:
The name of this module might be confused with https://pypi.org/project/constants/
Could we just call it `client_info.py`?

This feels like app name/version should canonically be defined in a `setup.py` for this tool.

This is important because we can begin a precedence / policy "all tools must maintain a version in a setup.py" to help standardize this.